web/webpack.config.ts

```diff
@@ -194,6 +195,9 @@ export default (env: {minimize?: boolean} = {}, argv: {mode?: string}): webpack.
             },
         },
         plugins: [
+            new DefinePlugin({
+                ZULIP_VERSION: JSON.stringify(process.env.ZULIP_VERSION || "development"),
+            }),
```

Comment:
@andersk I suspect you may prefer some other way of exposing this value that won't be a global -- do you have a recommended alternative approach?